### PR TITLE
Fixed LeftCurly

### PR DESCRIPTION
**Pull request by grzesiek2010**

### CODE REVIEW

File: src/org/javarosa/core/model/actions/SetValueAction.java [hunk 1]
```diff
@@ -78,7 +78,9 @@ public void processAction(FormDef model, TreeReference contextRef) {
 		}
 
 		AbstractTreeElement node = context.resolveReference(qualifiedReference);
-		if(node == null) { throw new NullPointerException("Target of TreeReference " + qualifiedReference.toString(true) +" could not be resolved!"); }
+		if(node == null) {
```
Comment (dcbriccetti) [inline src/org/javarosa/core/model/actions/SetValueAction.java]: If you’re improving formatting, how about a space after if?

File: src/org/javarosa/core/model/condition/EvaluationContext.java [hunk 2]
```diff
@@ -116,8 +116,12 @@ public void setOriginalContext(TreeReference ref) {
 	}
 
 	public TreeReference getOriginalContext() {
-		if (this.original == null) { return this.contextNode; }
-		else { return this.original; }
+		if (this.original == null) {
```
Comment (dcbriccetti) [inline src/org/javarosa/core/model/condition/EvaluationContext.java]: Is this the indentation amount we want?

File: src/org/javarosa/core/model/actions/SetValueAction.java (hunk 1 above)
Comment (grzesiek2010) [inline src/org/javarosa/core/model/actions/SetValueAction.java]: I've fixed only LeftCurly checkstyle. A space is another one and I can fix it in the future as a separate pr.

File: src/org/javarosa/core/model/condition/EvaluationContext.java (hunk 2 above)
Comment (grzesiek2010) [inline src/org/javarosa/core/model/condition/EvaluationContext.java]: As above it's only `LeftCurly` checkstyle. I don't care about anything else here.

Comment (dcbriccetti) [inline src/org/javarosa/core/model/condition/EvaluationContext.java]: Oh, I see. Thanks.

**Comment by lognaturel** [comment]
@dcbriccetti Looks like the current open pull requests all require longer discussion that probably won't conclude in the short term. Would now be an ok time for @grzesiek2010 to add a few this check and a few others so that we have fewer style inconsistencies to work around?

**Comment by dcbriccetti** [comment]
>Would now be an ok time

Sure.

**Pull request closed without merging**